statick_tool/statick.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import os
 
+from six import iteritems
```

Comment:
I could not figure out how to get Python 2/3 compatibility and fix the warning without bringing in this new module. I think it is a really common module to use in Python, so I'm okay with it.